src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -525,6 +535,60 @@ class PickPage extends Component {
       .catch(() => { this.props.hideSpinner(); });
   }
 
+  // Returns indexes of rows with quantity picked 0, and without subitems.
+  getRowsWithEmptyPicks(pickPageItems) {
+    return pickPageItems.reduce((acc, item, index) => {
+      // When the quantity picked is equal to 0 we have to check its subitems,
+      // because the item can be edited to 0, not cleared.
+      if (!item.quantityPicked && !item.picklistItems.length) {
+        return [...acc, index + 1];
```

Comment:
what is the reason for `index + 1` ?
As I am reading a function and the comment we get a list of indexes where quantity is zero or empty and we don't have pickListItems, but we are appending an index of the next item :thinking:

Reply:
I think I am going to answer this myself...
 this is for user readability 
"The picked lot is missing at rows: 1,2,3 ..."
instead of
"The picked lot is missing at rows: 0, 1,2 ..."